index.php

```diff
@@ -236,6 +236,12 @@ public function checkForUpdate() {
 			$updateText = 'No update available.';
 		}
 
+		if ($this->updateAvailable && isset($response['autoupdater']) && !($response['autoupdater'] === 1 || $response['autoupdater'] === '1')) {
```

Comment:
Doesn't this evaluate to the same at the end?

`!($response['autoupdater'] === 1 || $response['autoupdater'] === '1')`